packages/falcon-magento2-api/src/infrastructure/number.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Try to parse provided value to Number.
+ * @param {object} x value to parse
+ * @return {Number|undefined} `Number` if can or `undefined` otherwise
```

Comment:
```suggestion
 * @returns {number|undefined} Number or `undefined` result
```

Reply:
@Borales should not be rather "`Number`" instead of "Number"?

```suggestion
* @returns {number|undefined} `Number` or `undefined` result
```

Reply:
done